aixplain/modules/pipeline/designer/nodes.py

```diff
@@ -288,11 +288,15 @@ def __init__(self, value: DataType, path: List[Union[Node, int]], operation: Ope
         self.operation = operation
         self.type = type
 
-        if not self.path:
-            raise ValueError("Path is not valid, should be a list of nodes")
+        # Path can be an empty list in case the user has a valid case
```

Comment:
Remove the comment.

Reply:
The comment is related with the commented section, why we need to remove?